code/controllers/subsystem/ghost_spawns.dm

```diff
@@ -0,0 +1,216 @@
+SUBSYSTEM_DEF(ghost_spawns)
+	name = "Ghost Spawns"
+	init_order = INIT_ORDER_EVENTS
+	flags = SS_BACKGROUND
+	wait = 1 SECONDS
+	runlevels = RUNLEVEL_GAME
+	offline_implications = "Ghosts will no longer be able to respawn as event mobs (Terror, Blob, etc..). Shuttle call recommended."
+
+	var/list/datum/candidate_poll/currently_polling // List of polls currently ongoing, to be checked on next fire()
+	var/polls_active = FALSE // Whether there are active polls or not
+	var/total_polls = 0 // Number of polls performed since the start
+	var/datum/candidate_poll/next_poll_to_finish // The poll that's closest to finishing
+
+/datum/controller/subsystem/ghost_spawns/fire()
```

Comment:
Minor styling, but `fire()` usually goes after `Initialize()` in the file, and if you had a recover, would go at the very bottom of the file